cardinal/world.go

```diff
@@ -382,6 +379,21 @@ func (w *World) startServer() {
 	}()
 }
 
+func (w *World) BuildQueryFields() []engine.FieldDetail {
```

Comment:
This should be in QueryManager

Reply:
If we move it onto manager the manager interface it loses access to world. 

We can pass it in as `w.BuildQueryFields(w)`  but this breaks the Provider pattern on server.

Reply:
It doesn't need access to world. GetRegisteredQueries is just calling the QueryManager anyway, you can just make a direct call to it from within querymanager [](edited-with-Axolo)

Reply:
missed that.